app/models/enterprise.rb

```diff
@@ -462,6 +455,17 @@ def devise_mailer
 
   private
 
+  def validate_white_label_logo_link
+    return if white_label_logo.blank?
```

Comment:
I was going to suggest this dependency isn't necessary, but then remembered the link input field is hidden when the checkbox is unchecked. 
So it's not fair on the user to validate something they can't fix. (Eg if we removed this check, then the validation rules change in the future, an old hidden value could cause an error that the user can't see).

So all good 👍